core/src/main/kotlin/com/malinskiy/marathon/report/summary/TestSummary.kt

```diff
@@ -1,10 +1,19 @@
 package com.malinskiy.marathon.report.summary
 
 import com.malinskiy.marathon.execution.TestResult
+import com.malinskiy.marathon.execution.TestStatus
 import com.malinskiy.marathon.test.Test
 
 data class TestSummary(
     val test: Test,
     val results: List<TestResult>,
     val batches: List<Batch>
-)
+) {
+
+    val isFlaky: Boolean by lazy {
+        val hasSuccessResult = results.any { it.status == TestStatus.PASSED }
+        val hasFailedResult = results.any { it.status == TestStatus.FAILURE }
```

Comment:
We don't care about proportion here? Like if we have 1 failure per 100 passes it will still be considered flaky?